src/pas/plugins/oidc/browser/view.py

```diff
@@ -191,7 +191,8 @@ def __call__(self):
             # 'state': session.get('end_session_state'),
             # TODO: ....
             # 'post_logout_redirect_uri': api.portal.get().absolute_url(),
-            "redirect_uri": redirect_uri,
```

Comment:
although deprecated, is it used by any OIDC compliant servers?

Perhaps it doesn't harm to keep it, right?

Reply:
If we keep it the error persists.

The solution maybe to create an option to use the old or new way.

Reply:
Then, if there are implentations out there that use one or the other, I would add an option in the plugin config, like the ones we have, and make the code conditional.

Reply:
News update, I was testing:
  - With legacy enabled in keycloak, the new code worked.
  - With legacy enabled in keycloak, the old code also works, as tested before.
  - With legacy disabled in keycloak, the new code works.
  - With legacy disabled in keycloak, the old code do not work.

So, for keycloak, it do not matter if we use the post_logout or redirect_uri if legacy is **enabled**. 

If legacy is disabled, this is the default since version 18 of keycloak according to this comment in starckoverflow: https://stackoverflow.com/a/72142887

So, with this change the problem will happen only if the person enables the legacy and the server runs with legacy disabled.

Reply:
Form with the new option.

![Captura de tela de 2023-07-14 16-34-15](https://github.com/collective/pas.plugins.oidc/assets/293174/3933ddd9-1f6b-4e52-a899-b3e90c36b3f1)

Reply:
Option disabled, using post_logout.

![Captura de tela de 2023-07-14 16-35-17](https://github.com/collective/pas.plugins.oidc/assets/293174/75b0da40-ac94-4ecb-a51a-0e597a4dda31)